src/Redux/Reducers/bookingDetailsSlice.js

```diff
@@ -1,29 +1,31 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
 import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+// eslint-disable-next-line import/no-extraneous-dependencies
 import axios from 'axios';
 
-export const fetchHouseById = createAsyncThunk('houses/fetchHouseById', async (id) => {
+export const fetchReservationById = createAsyncThunk('houses/fetchReservationById', async (id) => {
   const response = await axios.get(`http://localhost:4000/api/v1/houses/${id}`);
   return response.data;
 });
```

Comment:
This method is used to fetch details for a specific house using its id. Please do not use it for the reservations.

Reply:
Ok

Reply:
Do you mean it can be discarded or refactored?